eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -26,6 +26,76 @@ steps:
     inlineScript: |
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
+- task: CredScan@2
+  continueOnError: "false"
+  inputs:
+    toolMajorVersion: "V2"
+
+- task: SdtReport@1
+  displayName: "Generate Analysis Report"
+  continueOnError: "false"
+  inputs:
+    CredScan: true
+    ToolLogsNotFoundAction: "Standard"
+
+- task: PublishSecurityAnalysisLogs@2
```

Comment:
Do we need to publish the logs for PR builds?  If it's going to email the team, and maybe create a work item for the team to fix, this is too much for code that hasn't been merged into dev.

Reply:
Ok. Removed.